ckanext/validation/i18n/fr/LC_MESSAGES/ckanext-validation.po

```diff
@@ -826,9 +828,62 @@ msgstr ""
 "- Si l'erreur doit être ignoré, il faut désactiver la vérification "
 "`maximum-length-constraint` dans le {validator}."
 
+#: ckanext/validation/fanstatic/js/module-validation-report.js:86
+#TODO: add in FR translation
+msgid "Invalid Header for DataStore"
+msgstr ""
+
+#: ckanext/validation/fanstatic/js/module-validation-report.js:87
+#TODO: add in FR translation
+msgid ""
+"Column name {value} in column {column_number} is not valid for a "
+"DataStore header"
+msgstr ""
+
+#: ckanext/validation/fanstatic/js/module-validation-report.js:88
+#TODO: add in FR translation
+msgid ""
+"Column name is invalid for a DataStore header.\n"
+"\n"
+" How it could be resolved:\n"
+" - Remove any leading underscores('_') from the column name.\n"
+" - Remove any leading for trailing white space from the column name.\n"
+" - Remove any double quotes('\"') from the column name.\n"
+" - Make sure the column name is not blank."
+msgstr ""
+
+#: ckanext/validation/fanstatic/js/module-validation-report.js:94
+#TODO: add in FR translation
+msgid "Header Too Long for DataStore"
+msgstr ""
+
+#: ckanext/validation/fanstatic/js/module-validation-report.js:95
+#TODO: add in FR translation
+msgid ""
+"Column name {value} in column {column_number} is too long for a DataStore"
+" header"
+msgstr ""
+
+#: ckanext/validation/fanstatic/js/module-validation-report.js:96
+#TODO: add in FR translation
+msgid ""
+"Column name is too long for a DataStore header.\n"
+"\n"
+" How it could be resolved:\n"
+" - Make the column name at most 63 characters long."
+msgstr ""
+
+#: ckanext/validation/fanstatic/vendor/goodtables-ui/goodtables-ui.min.js:11
+msgid "{|}~])/,autolink:/^<(scheme:[^sx00-x1f<>]*|email)>/,url:r,tag:\"^comment|^</[a-zA-Z][\\w:-]*\\s*>|^<[a-zA-Z][\\w-]*(?:attribute)*?\\s*/?>|^<\\?[\\s\\S]*?\\?>|^<![a-zA-Z]+\\s[\\s\\S]*?>|^<!\\[CDATA\\[[\\s\\S]*?\\]\\]>\",link:/^!?[(label)](s*(href)(?:s+(title))?s*)/,reflink:/^!?[(label)][(?!s*])((?:\\[[]]?|[^[]\\])+)]/,nolink:/^!?[(?!s*])((?:[[^[]]*]|\\[[]]|[^[]])*)](?:[])?/,strong:/^__([^s_])__(?!_)|^**([^s*])**(?!*)|^__([^s][sS]*?[^s])__(?!_)|^**([^s][sS]*?[^s])**(?!*)/,em:/^_([^s_])_(?!_)|^*([^s*<[])*(?!*)|^_([^s<][sS]*?[^s_])_(?!_|[^spunctuation])|^_([^s_<][sS]*?[^s])_(?!_|[^spunctuation])|^*([^s<\"][sS]*?[^s*])*(?!*|[^spunctuation])|^*([^s*\"<[][sS]*?[^s])*(?!*)/,code:/^("
+msgstr ""
+
+#: ckanext/validation/fanstatic/vendor/goodtables-ui/goodtables-ui.min.js:11
```

Comment:
I guess babel choked on this line-noise.. No reason for it to be in the po/pot files

Reply:
@wardi hmm yeah, I see that the message extractors are targeting JS. But they really should not target vendor/js.

I have removed these manually from the pot and po file for now. And will figure out the message extractor to prevent vendor/js crawls tomorrow.